app.test.js

```diff
@@ -52,9 +52,48 @@ describe('get /api',()=>{
     .get('/api')
     .expect(200)
     .then(({body})=>{
-    expect(body).toEqual(data);
+    expect(body).toEqual(endpointsData);
     
     })
 })
 })
-    
\ No newline at end of file
+
+describe('GET /api/articles/:article_id',()=>{
+    test('should respond with 200 and the correct article for the id provided',()=>{
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body})=>{
+            expect(typeof body).toBe('object')
+            expect(body.article).toHaveProperty('author');
```

Comment:
It would be ideal to assert that we expect each property to have values attached to them as well